ouroboros-consensus-shelley/test/Test/ThreadNet/TxGen/Shelley.hs

```diff
@@ -11,16 +12,18 @@ module Test.ThreadNet.TxGen.Shelley
 import           Control.Monad.Except (runExcept)
 import           Crypto.Number.Generate (generateBetween, generateMax)
 import           Crypto.Random (MonadRandom)
+import qualified Data.Sequence as Seq
 
 import           Ouroboros.Consensus.Config
 import           Ouroboros.Consensus.Ledger.Abstract
 
 import qualified Shelley.Spec.Ledger.LedgerState as SL
 import qualified Shelley.Spec.Ledger.STS.Ledger as STS
+import qualified Shelley.Spec.Ledger.Tx as STx
```

Comment:
```suggestion
import qualified Shelley.Spec.Ledger.Tx as SL
```
(For consistency. If we want to have more specific qualifiers, which I wouldn't object  to, we should do it consistently throughout the Shelley code.)